extension/src/html/popup.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Chrome Extension</title>
+</head>
+
+<body>
+    <div id="react-target"></div>
+</body>
+
+</html>
```

Comment:
```suggestion
</html>

```
nit: add a new line at EOF